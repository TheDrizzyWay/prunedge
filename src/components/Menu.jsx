import React from 'react';

const Menu = () => {
    return (
        <div className="menu">
            <div className="item">
                <a className="active" href="/">
                    <i className="fas fa-home"></i> Dashboard
                </a>
            </div>
            <div className="item">
                <a href="/">
                    <i className="fas fa-bookmark"></i> Documents
                </a>
            </div>
            <div className="item">
                <a href="/">
                    <i className="fas fa-bell"></i> Activity
                </a> 
                <div className="badge">
                    <p>12</p>
                </div>
            </div>
            <div className="item">
                <a href="/">
                    <i class="fas fa-archive"></i> Archive
                </a>
            </div>
            <div className="item">
                <a href="/">
                    <i class="fas fa-cog"></i> Settings
                </a>
            </div>
            <div className="filler"></div>
            <div className="item item__logout">
                <a className="logout" href="/">
                    <i className="fas fa-arrow-circle-left"></i> Log Out
                </a>
            </div>
        </div>
    );
}

export default Menu;
