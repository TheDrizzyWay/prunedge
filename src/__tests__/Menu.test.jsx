import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Menu from '../components/Menu';

configure({ adapter: new Adapter() });

describe('Menu component', () => {
  it('should render menu component', () => {
    const wrapper = shallow(
      <Menu />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
