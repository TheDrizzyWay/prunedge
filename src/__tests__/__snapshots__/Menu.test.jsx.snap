// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component should render menu component 1`] = `ShallowWrapper {}`;
